frontend/js/src/playlists/components/PlaylistMenu.tsx

```diff
@@ -224,114 +224,108 @@ function PlaylistMenu({
   );
   const showAppleMusicExportButton = appleAuth;
   return (
-    <ul
+    <div
       className="dropdown-menu dropdown-menu-right"
       aria-labelledby="playlistOptionsDropdown"
     >
-      <li>
-        <a onClick={copyPlaylist} role="button" href="#">
-          <FontAwesomeIcon icon={faCopy as IconProp} />{" "}
-          {playlistID ? "Duplicate" : "Save to my playlists"}
-        </a>
-      </li>
+      <a
+        onClick={copyPlaylist}
+        role="button"
+        href="#"
+        className="dropdown-item"
+      >
+        <FontAwesomeIcon icon={faCopy as IconProp} />{" "}
+        {playlistID ? "Duplicate" : "Save to my playlists"}
+      </a>
       {isPlaylistOwner(playlist, currentUser) && (
         <>
-          <li role="separator" className="divider" />
-          <li>
-            <a
-              data-toggle="modal"
-              data-target="#CreateOrEditPlaylistModal"
-              role="button"
-              href="#"
-              onClick={() => {
-                NiceModal.show(CreateOrEditPlaylistModal, {
-                  playlist,
-                  coverArtGridOptions,
-                  currentCoverArt,
-                })
-                  // @ts-ignore
-                  .then((editedPlaylist: JSPFPlaylist) => {
-                    if (onPlaylistSaved) {
-                      onPlaylistSaved(editedPlaylist);
-                    }
-                  });
-              }}
-            >
-              <FontAwesomeIcon icon={faPen as IconProp} /> Edit
-            </a>
-          </li>
-          <li>
-            <a
-              data-toggle="modal"
-              data-target="#ConfirmPlaylistDeletionModal"
-              role="button"
-              href="#"
-              onClick={() => {
-                NiceModal.show(DeletePlaylistConfirmationModal, { playlist })
-                  // @ts-ignore
-                  .then((deletedPlaylist: JSPFPlaylist) => {
-                    if (onPlaylistDeleted) {
-                      onPlaylistDeleted(deletedPlaylist);
-                    }
-                  });
-              }}
-            >
-              <FontAwesomeIcon icon={faTrashAlt as IconProp} /> Delete
-            </a>
-          </li>
-        </>
-      )}
-      <li role="separator" className="divider" />
-      {showSpotifyExportButton && (
-        <li>
+          <div role="separator" className="dropdown-divider" />
```

Comment:
The css for those separators needs tweaking, they look invisible at the moment:
![image](https://github.com/user-attachments/assets/cc309980-b4f6-4e0b-b724-86149df6b5e8)

Border color should be `#e5e5e5` to replicate the existing design

Reply:
Fixed